x-pack/plugin/core/src/main/resources/searchable-snapshots-template.json

```diff
@@ -0,0 +1,31 @@
+{
+  "index_patterns": [
+    ".snapshots-${xpack.searchable_snapshots.template.version}"
+  ],
+  "priority": 100,
+  "version": ${xpack.searchable_snapshots.template.version},
+  "data_stream": {
+  },
+  "composed_of": [
+  ],
+  "template": {
+    "aliases": {
+      ".snapshots": {}
+    },
+    "settings": {
```

Comment:
Should `.snapshots` indices also have `index.hidden: true` set in their settings? (I can't remember right now whether dot-prefixed indices automatically are hidden)